landing-strip/core/src/main/java/com/novoda/landingstrip/Coordinates.java

```diff
@@ -0,0 +1,6 @@
+package com.novoda.landingstrip;
+
+public interface Coordinates {
```

Comment:
the coordinates are mutable because we don't want to be new'ing up objects in `onDraw`